database/migrations/2016_04_01_140748_create_invitations_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateInvitationsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('invitations', function (Blueprint $table) {
+            $table->increments('id');
+            $table->boolean('status');
```

Comment:
yup ticket has more than invitations. invitations are just accepted or rejected " according to mariam"